apps/web/src/locales/en/messages.json

```
@@ -5268,6 +5268,15 @@
   "required": {
     "message": "required"
   },
+  "characterMaximum": {
+    "message": "$MAX$ character maximum",
```

suggestion: change to plural

```suggestion
    "message": "$MAX$ characters maximum",
```

or maybe 

```suggestion
    "message": "$MAX$ character(s) maximum",
```

but I guess we'd never have 1 character as maximum

---

the plural sounds a little strange when I say it out loud. Any thoughts @bitwarden/dept-design?

---

@jlf0dev I agree. Let's leave it singular.